src/main/java/io/stargate/sgv2/jsonapi/service/operation/filters/table/BooleanTableFilter.java

```diff
@@ -0,0 +1,9 @@
+package io.stargate.sgv2.jsonapi.service.operation.filters.table;
+
+/** Filter to use any JSON number against a table column. */
```

Comment:
Is this correct? Name implies "Boolean", similarly type parameter -- not number?

Reply:
yes, thanks!